checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,309 @@
             "waf": "Operations",
             "service": "Firewall",
             "guid": "e960fc6b-4ab2-4db6-9609-3745135f9ffa",
-            "id": "D06.18",
+            "id": "D07.10",
             "ammp": true,
             "severity": "Important",
             "training": "https://learn.microsoft.com/learn/paths/secure-networking-infrastructure/",
             "link": "https://learn.microsoft.com/en-us/azure/firewall-manager/migrate-to-policy"
         },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Segmentation",
+            "text": "Use a /26 prefix for your Azure Firewall subnets.",
+            "waf": "Security",
+            "service": "Firewall",
+            "guid": "22d6419e-b627-4d95-9e7d-019fa759387f",
+            "id": "D07.11",
+            "ammp": true,
+            "severity": "High",
+            "graph": "resources | where type=='microsoft.network/virtualnetworks' | project id,subnets=properties.subnets | mv-expand subnets | project id, subnetName = subnets.name, subnetPrefix = subnets.properties.addressPrefix | extend subnetPrefixLength = split(subnetPrefix, '/')[1] | where subnetName == 'AzureFirewallSubnet' | extend compliant = (subnetPrefixLength == 26) | distinct id, compliant",
+            "link": "https://learn.microsoft.com/azure/firewall/firewall-faq#why-does-azure-firewall-need-a--26-subnet-size"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Optimise Rule Configuration and processing",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "828cec2e-af6c-40c2-8fa2-1b681ee63eb7",
+            "id": "D07.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall-manager/rule-hierarchy"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use IP Groups or IP prefixes to reduce number of IP table rules",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "0da83bb1-2f39-49af-b5c9-835fc455e3d1",
+            "id": "D07.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/ip-groups"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Avoid wildcards as sources for DNATS",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "c44c6f0e-1642-4a61-a17b-0922f835c93a",
+            "id": "D07.13",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/tutorial-firewall-dnat"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
```

Comment:
This needs more specific guidance in the list.  How could the customer tell if they are preventing this, as part of a review?

Reply:
Sorry - this comment was for the one below, related to SNAT port exhaustion.

Reply:
I've added clearer guidance